api/proto/teleport/legacy/types/types.proto

```diff
@@ -4210,11 +4210,13 @@ message LockSpecV2 {
     (gogoproto.nullable) = true,
     (gogoproto.jsontag) = "expires,omitempty"
   ];
-  google.protobuf.Timestamp CreatedOn = 4 [
+  // CreatedAt is the date time that the lock was created.
+  google.protobuf.Timestamp CreatedAt = 4 [
     (gogoproto.stdtime) = true,
-    (gogoproto.nullable) = true,
-    (gogoproto.jsontag) = "created_on,omitempty"
+    (gogoproto.nullable) = false,
```

Comment:
```suggestion
```

Reply:
I tried that but the generated file still had the type as a pointer. I had to explicitly set it as false.